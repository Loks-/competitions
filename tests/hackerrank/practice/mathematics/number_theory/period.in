4
0 0 13
1 0 7
3 0 10007
1 1 19
