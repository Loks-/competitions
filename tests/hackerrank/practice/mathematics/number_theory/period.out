-1
1
5003
18
